translations/src/config.js

```diff
@@ -44,10 +44,7 @@ class Config {
     es: { rtl: false },
     sr: { rtl: false },
     ps: { rtl: true },
-    kmr: {
-      rtl: true,
-      additionalFont: 'lateef'
-    },
+    kmr: { rtl: false },
```

Comment:
Actually I would stick with `kmr`:
> Script subtags should only be used as part of a language tag when the script adds some useful distinguishing information to the tag. Usually this is because a language is written in more than one script or because the content has been transcribed into a script that is unusual to the language (so one might tag Russian transcribed into the Latin script with a tag such as ru-Latn).

Reply:
I think kmr or kmr-Latn is fine as there is a alias defined in the "one and only locale repository": https://github.com/unicode-org/cldr-json/blob/872e62d6ca2343412311520909ee0c1f78e93d99/cldr-json/cldr-core/supplemental/aliases.json#L838

 @klinzo I did not find hints that kmr can be written RTL. Where did you find that?

Reply:
https://de.wikipedia.org/wiki/Kurmandschi
> Kurmandschi wird seit den 1930er Jahren vorwiegend im kurdisch-lateinischen Alphabet geschrieben.

Yeah even though we could do that, I think it does not mean we should do it : D I mean as quoted w3.org says it should only include the script if it adds useful distinguishing information, which is not the case here imo as Kurmandschi is only written in latin.

Did I really say krm can be written in RTL? I don't think so :see_no_evil:

Reply:
So we decided to used `kmr` for northern Kurdish and `ckb-Latn` and `ckb-Arab` for the different Sorani versions

Reply:
Is `ckb-Arab` correct? I think it is a persian alphabet :thinking: 
https://de.wikipedia.org/wiki/Sorani#Verbreitung_und_Dialekte
> Zur Schreibung des Sorani wird meist die Persische Schrift mit Sonderzeichen verwendet

Reply:
I think there is no subtag for the persian alphabet explicitly. But as the persian and arab alphabet are almost similar I guess the tag is `ckb-Arab`. I also found an old issue in the firefox dev history where they say ckb-Arab is correct: https://bugzilla.mozilla.org/show_bug.cgi?id=580798#c5

Reply:
Seems correct, did not find one here: https://en.wikipedia.org/wiki/ISO_15924